src/SwiftUI/SwiftUI/SwiftUIExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+using Swift;
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	public unsafe static class SwiftUIExtensions
+	{
+		public static ModifiedOpacity<T> Opacity <T>(this T view, double opacity) where T : View
```

Comment:
Mono coding convention
```suggestion
		public static ModifiedOpacity<T> Opacity<T> (this T view, double opacity) where T : View
```

Reply:
I think we need to add a formatting file, as it is quite different in lots of places.